daycare/migrations/0005_image_data_migration.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 4.0.4 on 2022-04-11 19:07
+
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('daycare', '0004_static_daycare_users'),
+    ]
+
+    def generate_data(apps, schema_editor):
+        from daycare.models import DayCare, Image
+
+        DAYCARE_IMAGE_LIST = [
+            "https://i.pinimg.com/originals/83/c8/56/83c856787dec2ebb9fec486242f9e1d6.jpg",
+            "https://reddogpetresort.com/wp-content/uploads/2020/05/IMG_1701.jpg",
+            "https://www.crowdfunder.co.uk/uploads/projects/26396.jpg",
+            "http://savvyvirtualassistant.com/BFBB/wp-content/uploads/2012/03/290055_482309965129233_132653009_o.jpg",
+            "https://www.theboneadventure.com/images/bone-backyard/greenhouse.jpg",
+            "https://www.theboneadventure.com/images/bone-home/bone-home04.jpg",
+            "https://www.theboneadventure.com/images/600x350-Image-OliveBranch.jpg",
+            "https://www.thepoochpatio.com/images/home-featured.jpg",
+            "https://boston.cbslocal.com/wp-content/uploads/sites/3859903/2014/01/commondog.jpg",
+            "https://i.pinimg.com/originals/51/47/2f/51472f3314975b24b58e999c8878a0b1.jpg",
+            "https://i.pinimg.com/originals/36/c0/a9/36c0a940cc5904337eb949a6634bccf3.jpg",
+            "https://www.flyingfurintheburg.com/wp-content/uploads/small-dog-daycare-1.jpg",
+            "https://www.flyingfurintheburg.com/wp-content/uploads/small-dog-daycare-2.jpg",
+            "https://www.flyingfurintheburg.com/wp-content/uploads/small-dog-daycare-3.jpg",
+            "https://www.flyingfurintheburg.com/wp-content/uploads/small-dog-daycare-4.jpg",
+            "https://www.flyingfurintheburg.com/wp-content/uploads/small-dog-daycare-5.jpg"
+        ]
+
+        static_daycare_id = []
+        static_daycare_images = []
+
+        for daycare in DayCare.objects.all():
+            if 'static' in daycare.user.username:
+                static_daycare_id.append(daycare)
```

Comment:
You can use [list comprehension](https://www.w3schools.com/python/python_lists_comprehension.asp) instead, making it more pythonic, instead of declaring an empty list first, c-style.